docs/src/IconButton.doc.js

```diff
@@ -149,6 +155,46 @@ function A11yEx() {
   />
 );
 
+card(
+  <Example
+    id="accessibility-disclosure"
+    description={`
+      We want to make sure every button on the page is unique when being read by screenreader.
+      - \`accessibilityControls\` allows us to specify the reference for an associated content (i.e. Accordion panel) which is controlled by this icon button.
```

Comment:
not super familiar, but it looks like `accessibilityControls` should match the `id` prop of the associated content/component. If so, could we make this explicit in the description?

Reply:
Great idea, doing it for the 3 docs (`IconButton`, `Button` and `Touchable`)